File: Scaler/prime_number.py
number=int(input("Enter the number for check prime or not "))
i=2
count = 0
while(i<=number//2):
    if(number%i==0):
        count = count + 1
        break
    i=i+1

if (count==0 and number!=1):
    print('{} is a prime number'.format(number) )
else:
    print(f'{number} is not a prime number')    

File: Scaler/PrimeNumber.java
package Scaler;

import java.util.Scanner;

public class PrimeNumber {
    private static Scanner sc = new Scanner(System.in);

    public static void main(String[] args) {
        System.out.print("Enter the number to check Prime or not: ");
        int num = sc.nextInt();
        int count = 0;
        int i = 2;

        while (i <= num / 2) {
            if (num % i == 0) {
                count += 1;
                break;
            }
            i += 1;
        }
        if (count == 0 && num != 1) {
            System.out.println(num + " is the prime number");
        } else {
            System.out.println(num + " is not the prime number");
        }

    }

}
